riff-raff/app/assets/javascripts/notifications.coffee

```diff
@@ -0,0 +1,17 @@
+Notification.requestPermission()
+
+intervalId = null
+
+checkStatus = () ->
+  if $('[data-run-state]').length != 0
+    buildName = window.riffraff.buildName
+    switch $('[data-run-state]').data('run-state')
+      when 'Failed' then new Notification('Riffraff', {body: 'Deployment of ' + buildName + ' failed!'})
+      when 'Completed' then new Notification('Riffraff', {body: 'Deployment of ' + buildName + ' is finished'})
+    disableCheck()
```

Comment:
Since you set data-run-state on NotRunning this will disable the check immediately when a deploy is waiting to run.